lib/open_project/costs/engine.rb

```diff
@@ -26,10 +26,13 @@ class Engine < ::Rails::Engine
     include OpenProject::Plugins::ActsAsOpEngine
 
     register 'openproject-costs',
-             :author_url => 'http://finn.de',
-             :requires_openproject => '>= 4.0.0',
-             :settings =>  { :default => { 'costs_currency' => 'EUR','costs_currency_format' => '%n %u' },
-             :partial => 'settings/openproject_costs' } do
+             author_url: 'http://finn.de',
+             requires_openproject: '>= 4.0.0',
+             settings: {
+               default: { 'costs_currency' => 'EUR','costs_currency_format' => '%n %u' },
```

Comment:
Line is too long. [89/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.